.github/scripts/check-cli-binaries.sh

```diff
@@ -36,8 +48,8 @@ case $(uname | tr '[:upper:]' '[:lower:]') in
     ;;
 esac
 echo "Assumed OS: $os_name"
-
-if [ -z ${PRISMA_FORCE_NAPI+x} ]; then
+echo $CLI_QUERY_ENGINE_TYPE == "binary"
+if [ $CLI_QUERY_ENGINE_TYPE == "binary" ]; then
   echo "Node-API: Disabled"
```

Comment:
Maybe renaming these debug logs would make more sense?

Reply:
Rename to what and why?